README.md

```diff
@@ -46,18 +92,6 @@ changes since the 0.3 series.
 
 For more details, see the [changelog](CHANGELOG.md).
 
-### Compatibility shims
-
-**As of now there is no compatibility shim between Rand 0.4 and 0.5.**
-It is also not entirely obvious how to make one due to the large differences
-between the two versions, although it would be possible to implement the new
-`RngCore` for any implementation of the old `Rng` (or vice-versa; unfortunately
-not both as that would result in circular implementation). If we implement a
-compatibility shim it will be optional (opt-in via a feature).
-
-There is a compatibility shim from 0.3 to 0.4 forcibly upgrading all Rand 0.3
-users; this is largely due to the small differences between the two versions.
-
 ### Rust version requirements
 
 The 0.5 release of Rand will require **Rustc version 1.22 or greater**.
```

Comment:
Might as well get this right for the actual release now? `will require` → `requires`; also maybe remove the below?